src/connector/mysql/error.rs

```diff
@@ -33,6 +33,17 @@ impl From<my::Error> for Error {
 
                 builder.build()
             }
+            my::Error::Server(ServerError { ref message, code, .. }) if code == 1317 => {
```

Comment:
Sooo my undenstanding is that the error codes are different between MySQL and Vitess. Are we sure they don't overlap?